src/window_numberinput.cpp

```diff
@@ -84,8 +85,9 @@ int Window_NumberInput::GetMaxDigits() {
 }
 
 void Window_NumberInput::SetMaxDigits(int idigits_max) {
-	// Only accepts values between 1 and 6 (or 7) as RPG2K (or RPG2k3)
-	int top = Player::IsRPG2k() ? 6 : 7;
+	// At least 7 digits because of gold input in debug scene
+	// (free space and 6 digits for the gold value)
+	int top = std::max(7, digits_max);
```

Comment:
>  Add settings for minimum and maximum variable values 
this looks wrong because digits_max is updated by the param idigits_max afterwards

Reply:
Fixed.